src/pages/blog.tsx

```diff
@@ -0,0 +1,101 @@
+import { Container, Divider, Flex, Text, chakra } from '@chakra-ui/react';
+import { NextPage } from 'next';
+import Image from 'next/image';
+import React from 'react';
+import veltBusMan from '../assets/man-with-bus-background.jpg';
+import veltBoss from '../assets/velt-boss.jpg';
+import { Layout } from '../components/layout';
+
+export const Hero = chakra(Image, {
+  baseStyle: {
+    objectFit: 'cover',
+    width: '100%',
+    maxWidth: 'container.xl',
+    borderRadius: '2xl'
+  },
+  shouldForwardProp: prop =>
+    ['width', 'height', 'src', 'alt', 'fill', 'quality'].includes(prop),
+});
+
+const Blog: NextPage = () => (
+  <Layout>
+    <Container p="l">
+      <Hero
+        src={veltBoss}
+        alt="Picture of the boss in Velt"
+        quality={100}
+        placeholder="blur"
+      />
+      <Flex flexDirection="column" pt="m" gap="l">
```

Comment:
I'm not sure we need this. Seeing as all the child `<Flex />`'es are setting some form of padding value. We're using this only to achieve the `gap` between elements, right?

Reply:
Ah, yeah we have the Container around it regardless. The idea was to have it all in a Flex to make the transition to being CRM-based later easier, but I guess the Container does the trick.